test/reactive-insert.spec.ts

```diff
@@ -1,16 +1,27 @@
 /**
  * @jest-environment jsdom-latest
  */
-import { ReplaySubject } from 'rxjs';
+import { ReplaySubject, Subscription } from 'rxjs';
 import { horo } from '../src/horo';
+import { Component } from '../src/insertions/insertions';
 
 describe('Reactive Insert RxJS', () => {
+    let component: Component;
     const subject = new ReplaySubject<string>();
+
+    let spy: jasmine.Spy;
+    const originalSubscribe = subject.subscribe.bind(subject);
+    const newSubscribe = (...args: any[]): Subscription => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/lroskoshin/horo/pullRequest?prid=7435565)
```suggestion
    const newSubscribe = (...args: unknown[]): Subscription => {
```